client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx

```diff
@@ -663,8 +669,8 @@ export const ApplicationsTableAnalyze: React.FC = () => {
                       modifier="truncate"
                       {...getTdProps({ columnKey: "analysis" })}
                     >
-                      <IconedStatus
-                        preset={application.review ? "Completed" : "NotStarted"}
+                      <ApplicationAnalysisStatus
+                        state={getTaskState(application)}
```

Comment:
could do this and remove the getTaskState

```suggestion
                        state={ getTask(application)?.state || "No task" }
```

Reply:
@gitdallas, much slicker !!!